test/Vagrantfile

```diff
@@ -6,8 +6,14 @@ $JOB_NAME = ENV['JOB_BASE_NAME'] || "LOCAL"
 $K8S_VERSION = ENV['K8S_VERSION'] || "1.8"
 $K8S_NODES = (ENV['K8S_NODES'] || "2").to_i
 
-$SERVER_BOX= "cilium/ginkgo"
-$SERVER_VERSION="0.0.4"
+$distribution = ENV['DISTRIBUTION']
```

Comment:
Hi, 

Let me suggest a new way in the test/Vagrantfile: 

I would do something like: 

```
$DISTRIBUTION = ENV['DISTRIBUTION'] || ubuntu
... 
server.vm.box = cilium/$DISTRIBUTION
server.vm.box_version = $SERVER_VERSION
```

The tty line is needed.

On the other hand, both version will be the same, build on packer at the same
time, so will be the same, and are super important for us

Reply:
>On the other hand, both version will be the same, build on packer at the same
time, so will be the same, and are super important for us

Sorry, but I don't understand this sentence. What version you're referring to? `SERVER_VERSION`? They seem to be different for Ubuntu and openSUSE:

https://app.vagrantup.com/cilium/boxes/ubuntu (the newest is v0)
https://app.vagrantup.com/cilium/boxes/opensuse (the newest is v1, the oldest is v0.0.1)

Can't we just not define the box version and rely on the newest boxes which are available?

If for some reason (which I'm possibly not aware about) we need to define the server version, then we probably need to define the version for each distro - and keep it in some dictionary, hash table or any kind of data struct we can simply keep in the Vagrantfile.

Reply:
Also, your proposed code with `$DISTRIBUTION = ENV['DISTRIBUTION'] || ubuntu` looks simpler, but it will not give an explicit error if user will make a mistake in the distro name.

Reply:
Also, the tty line does **not** work on any distro. It seems that it worked for the older Ubuntu releases, but not for 16.10. The `/root/.profile` file just doesn't exist.

Reply:
Hi, 

The version of the boxes will be fixed today. Both servers will have the same version.  PR 8 in packer-ci-build ;-) 

Related with the version, no we don't want to use the latest, the idea is to have a specific version and be sure that we always use this. In case of a change in a base box, maybe we need to do some work in cilium, so we don't use that until a new PR that updates the version. 

Around the DISTRIBUTION,  if is not correct, the vagrant will give an error that the box does not exit. In the other hand, that options should be documented in the `docs/contributing.rst`. The idea is to have the vagrantfile as clean as possible.